control_mux:
  ros__parameters:
    absolute_max_speed: 0.75
    ai_topic: "/ai/cmd"
    joy_topic: "/joy"
    hw_interface_topic: "/hw/cmd"

